src/processing/vFramer/include/vDraw.h

```diff
@@ -125,6 +131,34 @@ class accDraw : public vDraw {
 
 };
 
+class rasterDraw : public vDraw {
+
+protected:
+    #define XLIMIT 304          //array needs constants
+    #define YLIMIT 240
+
+    double pixelStorage[YLIMIT][XLIMIT];
```

Comment:
there should be no need to add extra storage. plot directly from the events given to you